en/middleware/micrortps.md

```diff
@@ -1,34 +1,39 @@
 # RTPS/ROS2 Interface: PX4-FastRTPS Bridge
 
-The *PX4-FastRTPS Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *Fast RTPS* applications (including those built over the ROS2/ROS frameworks).
+The *PX4-FastRTPS Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *Fast RTPS* applications (including those built using the ROS2/ROS frameworks).
 
 > **Note** RTPS is the underlying protocol of the Object Management Group's (OMG) Data Distribution Service (DDS) standard. 
   It aims to enable scalable, real-time, dependable, high-performance and inter-operable data communication using the publish/subscribe pattern. *Fast RTPS* is a very lightweight cross-platform implementation of the latest version of the RTPS protocol and a minimum DDS API.
 
-RTPS has been adopted as the middleware for the ROS2 (Robot Operating System). 
+RTPS has been adopted as the middleware for the ROS2 (Robot Operating System).
 The bridge allows us to better integrate with ROS2, making it easy to share sensor values, commands, and other vehicle information.
 
-This topic describes the bridge architecture, how it is compiled, and how to:
-1. Write a simple *Fast RTPS* application to subscribe to PX4 changes;
-1. For ROS2/ROS1 applications, understand how the `px4_ros_com` package works and how to use it to bridge ROS nodes with PX4.
+This topic describes the RTPS bridge architecture (and how it is used within the ROS2/ROS application pipeline).
+It also shows how to compile needed code to:
+1. Write a simple *Fast RTPS* application to subscribe to PX4 changes
+1. Connect ROS2 and ROS1 nodes with PX4 (via the RTPS Bridge, and using the `px4_ros_com` package)
```

Comment:
Note, this changed. Basically you can now read both points separately and they both make sense.